source/manual/government-changes.html.md

```diff
@@ -109,3 +109,48 @@ we can run a rake task:
 
 This will end all ministerial roles except the Prime Minister. You might have to
 do that one manually.
+
+## Machinery of Government Changes
+
+Following machinery of government changes, developers can help content
+designers make bulk changes to documents and users.
+
+### Bulk changing users
+
+There is a Rake task in Signon to change the organisation that a user belongs
+to in bulk:
+[`data_hygiene:bulk_update_organisation[csv_file]`][signon-bulk-update-organisation].
+
+It accepts a CSV file should have at least the following columns (in any
+order):
+
+- `Old email` - The old email address of the user.
+- `New email` - The new email address of the user.
+- `New organisation` - The slug of the new organisations for the user.
+
+If you want to bulk move some users across to a different organisation, follow
+the CSV file guidance and then Platform Health can do this for you.
```

Comment:
Good catch, this line shouldn't actually be here, I accidentally copied it from the internal guidance that the content team have.

Reply:
To answer your question, anyone can perform these Rake tasks and hopefully this documentation means future requests like this will go straight to 2nd line.